Header header
string state
float32 distance
uint32 position1_x
uint32 position1_y
uint32 position2_x
uint32 position2_y

